stock_picking_report_valued_sale_mrp/models/stock_move_line.py

```diff
@@ -0,0 +1,90 @@
+# Copyright 2020 Tecnativa - David Vidal
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl).
+from odoo import api, fields, models
+
+
+class StockMoveLine(models.Model):
+    _inherit = 'stock.move.line'
+
+    phantom_product_id = fields.Many2one(
+        comodel_name="product.product",
+        compute="_compute_phantom_product_id",
+        compute_sudo=True,
+        string="Kit product",
+        readonly=True,
+    )
+    phantom_line = fields.Boolean(
+        compute="_compute_sale_order_line_fields",
+        compute_sudo=True,
+        default=False,
+    )
+    phantom_dlvd_qty = fields.Float(
```

Comment:
Don't use such cryptic abbreviates:
```suggestion
    phantom_delivered_qty = fields.Float(
```